docs/imagen.md

```diff
@@ -0,0 +1,188 @@
+```
+
+Podemos ver que, al tener que instalar el lenguaje de Go con todas sus dependencias, el tamaño de la imagen de **archlinux** se duplica, mientras que el de **alpine** solo aumenta un poco debido a que solo
+ha sido necesario instalar *just*.
+
+Veamos ahora la velocidad a la que se ejecutan los tests. Para ello, utilizamos el comando `time`:
+
+Para **archlinux**:
+
+```Bash
+diagmatrix@gibbon ~/git/maybe-better-maybe-worse $ time docker run -u 1001 -t -v `pwd`:/app/test diagmatrix/maybe-better-maybe-worse:archlinux
+Ejecutando los tests...
+go test -v ./tests/
+=== RUN   TestMemberTypeFromString
+--- PASS: TestMemberTypeFromString (0.00s)
+=== RUN   TestMemberTypeFromStringInvalid
+--- PASS: TestMemberTypeFromStringInvalid (0.00s)
+=== RUN   TestParseTimeTable
+--- PASS: TestParseTimeTable (0.00s)
+=== RUN   TestParseTimeTableInvalidFile
+--- PASS: TestParseTimeTableInvalidFile (0.00s)
+=== RUN   TestParseTimeTableInvalidHeader
+--- PASS: TestParseTimeTableInvalidHeader (0.00s)
+=== RUN   TestParseTimeTableInvalidAvailability
+--- PASS: TestParseTimeTableInvalidAvailability (0.00s)
+=== RUN   TestParseTechnicalGuide
+--- PASS: TestParseTechnicalGuide (0.00s)
+=== RUN   TestParseTechnicalGuideInvalidFile
+--- PASS: TestParseTechnicalGuideInvalidFile (0.00s)
+=== RUN   TestParseTechnicalGuideInvalidHeader
+--- PASS: TestParseTechnicalGuideInvalidHeader (0.00s)
+=== RUN   TestParseTechnicalGuideInvalidCastCrew
+--- PASS: TestParseTechnicalGuideInvalidCastCrew (0.00s)
+PASS
+ok  	github.com/diagmatrix/maybe-better-maybe-worse/tests	0.003s
+
+real	0m26,353s
+user	0m0,011s
+sys	0m0,011s
+```
+
+Para **alpine**:
+
+```Bash
+diagmatrix@gibbon ~/git/maybe-better-maybe-worse $ time docker run -u 1001 -t -v `pwd`:/app/test diagmatrix/maybe-better-maybe-worse:alpine
+Ejecutando los tests...
+go test -v ./tests/
+=== RUN   TestMemberTypeFromString
+--- PASS: TestMemberTypeFromString (0.00s)
+=== RUN   TestMemberTypeFromStringInvalid
+--- PASS: TestMemberTypeFromStringInvalid (0.00s)
+=== RUN   TestParseTimeTable
+--- PASS: TestParseTimeTable (0.00s)
+=== RUN   TestParseTimeTableInvalidFile
+--- PASS: TestParseTimeTableInvalidFile (0.00s)
+=== RUN   TestParseTimeTableInvalidHeader
+--- PASS: TestParseTimeTableInvalidHeader (0.00s)
+=== RUN   TestParseTimeTableInvalidAvailability
+--- PASS: TestParseTimeTableInvalidAvailability (0.00s)
+=== RUN   TestParseTechnicalGuide
+--- PASS: TestParseTechnicalGuide (0.00s)
+=== RUN   TestParseTechnicalGuideInvalidFile
+--- PASS: TestParseTechnicalGuideInvalidFile (0.00s)
+=== RUN   TestParseTechnicalGuideInvalidHeader
+--- PASS: TestParseTechnicalGuideInvalidHeader (0.00s)
+=== RUN   TestParseTechnicalGuideInvalidCastCrew
+--- PASS: TestParseTechnicalGuideInvalidCastCrew (0.00s)
+PASS
+ok  	github.com/diagmatrix/maybe-better-maybe-worse/tests	0.003s
+
+real	0m24,478s
+user	0m0,009s
+sys	0m0,010s
+```
+
+Podemos comprobar que en tiempo de ejecución casi idénticos.
```

Comment:
Como te he dicho en mi comentario, es complicado que en algo que tarda 3 milisegundos se encuentre una diferencia.

Reply:
¿Elimino entonces esta comprobación?

Reply:
Tú verás si te hace falta. Lo que yo buscaba con el comentario no es que hicieras la comprobación, sino que entendieras la diferencia entre los contenedores basados en Alpine y los basados en otras distros, y si realmente esa diferencia existe en el caso de Go. Alpine usa `musl' mientras que otros usan `libc`. Pero... tendrás que mirar como se construye el contenedor para ver si go usa realmente `musl` o realmente usa `libc` (que se puede instalar en Alpine).
Aparte de la biblioteca básica, puede haber otros factores que influyan en la velocidad relacionados por ejemplo con el scheduler, como se manejan las hebras y demás. No creo que ninguno tenga influencia en este caso.
TL;DR: Aparte que con algo que tarda 3 milisegundos es difícil que haya alguna diferencia entre diferentes sistemas (la teoría de ISE debería haberos explicado algo durante el benchmarking) lo que realmente perseguía yo con mi comentario es que entendáis las diferencias que hay entre diferentes imágenes base, aparte del tamaño.

Reply:
Mis disculpas. Creo que no tiene sentido ponerlo, y miraré en que se basa go

Reply:
Solucionado aquí [3be7b2f](https://github.com/diagmatrix/maybe-better-maybe-worse/pull/29/commits/3be7b2f4d4289b103295648749adb7b8c9005824)